master/stream.py

```diff
@@ -0,0 +1,89 @@
+import time
```

Comment:
question: another place where I'm wondering if we should include or not.

Reply:
I'm a proponent for keeping this in some capacity, but it might make more sense as a confluence page + sharepoint files; especially while the streamable types are limited

Reply:
I created a quick confluence page walking through the basic steps of running the current demo, I think we're good to not include it in this PR

Reply:
Also I'm working on the official python client, which will be much more thorough anyway.